spec/models/task_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  subject { described_class.new(title: 'Task #1') }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  it 'is invalid without a title' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  context 'save Task to database' do
+    it 'persists' do
+      expect do
```

Comment:
this is great, a couple thoughts:

instead of a `do end` you can do this in one line:

`expect { subject }.to change { Task.count }`

I would also probably say you would want to add on the following since deleting the record would also satisfy the existing test:

`expect { subject }.to change { Task.count }.by(1)`

Reply:
i would also say this should probably be just inside of the context of having valid attributes, since we would expect the opposite behavior